package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hyperview",
-  "version": "0.45.0",
```

Comment:
You don't need to update this version number, we do this automatically when pushing a release to npm (by running `yarn release:minor`). But that should wait after this code is approved and merged.

Reply:
reverted